app/aurelia/src/client/index.ts

```diff
@@ -0,0 +1,7 @@
+export * from './preview';
+
+// tsc wants to use NodeModule instead of WebpackModule
```

Comment:
Should be fixable by installing @types/webpack-env ?

Reply:
Added and it didn't complain